src/recipes.jl

```diff
@@ -1534,3 +1546,5 @@ julia> areaplot(1:3, [1 2 3; 7 8 9; 4 5 6], seriescolor = [:red :green :blue], f
         end
     end
 end
+
+is_3d(::Type{Val{:mesh3d}}) = true
```

Comment:
This should extend `RecipesPipeline.is3d`

Reply:
Should I open a separate pull request for RecipesPipeline for this then?

Reply:
yes, that would be good